torch/profiler/profiler.py

```diff
@@ -241,16 +397,45 @@ def __init__(
             self.schedule = _default_schedule_fn
             self.record_steps = False
         self.on_trace_ready = on_trace_ready
-        self.record_shapes = record_shapes
-        self.with_flops = with_flops
-        self.profile_memory = profile_memory
-        self.with_stack = with_stack
-        self.with_modules = with_modules
         self.step_num = 0
         self.current_action = self.schedule(self.step_num)
-        self.profiler: Optional[prof.profile] = None
         self.step_rec_fn: Optional[prof.record_function] = None
```

Comment:
this has been defined in the kineto_profiler class

Reply:
good catch